activities/Gridpaint.activity/js/gridpaint.js

```diff
@@ -119,21 +119,43 @@ function handleButton(y){
 //
 /////////////////////////
 
-function hittestInit(){
-	hitbuffer.style.visibility = 'hidden';
-	var ctx = hitbuffer.getContext('2d');
-	ctx.save();
-	ctx.translate(140, 20);
-	ctx.scale(scale, scale);
-	for(var i=0;i<shapes.length;i++){	
-		var low = hexdigit(i&0xf);
-		var mid = hexdigit((i>>4)&0xf);
-		var high = hexdigit((i>>8)&0xf);
-		ctx.fillStyle = '#'+high+'F'+mid+'F'+low+'F';
-		shapePath(i, ctx, true);
-	  ctx.fill();		
+var useragent = navigator.userAgent.toLowerCase();
+if (useragent.indexOf('chrome') == -1) {
```

Comment:
Hmmm. Not a big fan of this idea to differentiate the whole process.

Reply:
Can I know how did you calculate the value for translate (140,20) So that I can calculate common.

Reply:
Unfortunately I don't know. Was there from the beginning.